### DISCUSSION TITLE
Rename files-Question

### DISCUSSION THREAD

**Discussion by Tvirin** [rfjakob/gocryptfs#657] [Q&A] [answered]
When the name of the encrypted file is changed from file manager, it is no longer displayed in the application. This can change someone and I lose access to the files.

Is there a way to reopen the file with the changed name?

*_Basically this is happening on android, here I was referred by a dev from DroidFS but I think it's the same for gocryptfs_

**Comment by rfjakob** [comment, marked as answer]
Yes you can mount with option `-badname '*'`. Then the renamed files will be showed as `xyz.GOCRYPTFS_BAD_NAME`.

However, I would advise to not mess with the encrypted files in the file manager.
